src/bench/coin_selection.cpp

```diff
@@ -57,4 +61,47 @@ static void CoinSelection(benchmark::State& state)
     }
 }
 
+typedef std::set<CInputCoin> CoinSet;
```

Comment:
nit: C++ code should prefer `using` rather than `typedef` (not important here, but it's more flexible with templated types). Syntax would be:

```c++
using CoinSet = std::set<CInputCoin>;
```

Reply:
I would prefer to continue to use what is currently done in the codebase unless there we agreed to change the style.